web-bluetooth/device-info.html

```diff
@@ -0,0 +1,47 @@
+---
+feature_name: Device Info Web Bluetooth
+chrome_version: 45
+feature_id: 5264933985976320
+---
+
+<p>The <a href="https://github.com/WebBluetoothCG/web-bluetooth">Web Bluetooth API</a> discovers and communicates with devices over the Bluetooth 4 wireless standard using the Generic Attribute Profile (GATT). It is currently only partially implemented in Chrome OS behind the experimental flag <code>chrome://flags/#enable-web-bluetooth</code>.</p>
+
+<p>This sample illustrates the use of the Web Bluetooth API to retrieve basic
+device information from a nearby Bluetooth Low Energy Device.</p>
+
+<button>Get Bluetooth Device Info</button>
+{% capture initial_output_content %}
+{% endcapture %}
+{% include output_helper.html initial_output_content=initial_output_content %}
+
+{% capture js %}
+function onButtonClick() {
+
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice({filters:[{services:['generic_access']}]})
+  .then(device => {
+    log('> Device Name:       ' + device.name);
+    log('> Device InstanceID: ' + device.instanceID);
+    log('> Device Paired:     ' + device.paired);
+    log('> Device Class:      ' + device.deviceClass);
+    log('> Device UUIDs:      ' + device.uuids.join('\n'+'\x20'.repeat(21)));
+  })
+  .catch(error => {
+    log('Argh! ' + error);
+  });
+};
+{% endcapture %}
+{% include js_snippet.html js=js %}
+
+<script>
+document.querySelector('button').addEventListener('click', function() {
+  document.querySelector('#log').textContent = '';
```

Comment:
You might find it easier to use [`ChromeSamples.setStatus()`](https://github.com/GoogleChrome/samples/blob/gh-pages/_includes/output_helper.html#L11) here and elsewhere for reporting this error condition, instead of `ChromeSamples.log()`. It would save you the trouble of clearing out the `#log.textContent`.